cccore/Program.cs

```diff
@@ -763,5 +782,35 @@ public static void filter(HttpClient httpClient, string creditcoinUrl, List<stri
                 }
             }
         }
+
+        public static bool TryFetchProtobuf(HttpClient httpClient, string creditcoinUrl, List<string> ret, string address, out byte[] protobuf)
```

Comment:
Would be good to have a more informative name or description. The function not only tries to fetch any protobuf, it fetches raw data from a state address as Base64. The buffer is then parsed into a very particular protobuf, "Address", outside of the scope of the function.

Reply:
Let's assume this is the standard way to retrieve serialized items from a state address; a protobuf.